src/cartesian/ErrorBar.tsx

```diff
@@ -49,6 +50,9 @@ export type Props = SVGProps<SVGLineElement> & ErrorBarProps;
 export function ErrorBar(props: Props) {
   const { offset, layout, width, dataKey, data, dataPointFormatter, xAxis, yAxis, ...others } = props;
   const svgProps = filterProps(others);
+
+  invariant(!(props.direction === 'x' && xAxis.type !== 'number'), 'Errorbar requires Axis type to be "number".');
```

Comment:
can you make the text `ErrorBar` to match the component casing?

Reply:
sure will do this, also I did test the variant manually it throws when ErrorBar direction is x but a non number type is set to XAxis.